dev-util/pycharm-community/pycharm-community-2024.3.2.ebuild

```diff
@@ -0,0 +1,202 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop readme.gentoo-r1 wrapper xdg
+
+DESCRIPTION="Intelligent Python IDE with unique code assistance and analysis"
+
+# NOTE upstream release info
+# https://data.services.jetbrains.com/products?code=PCP,PCC&release.type=release
+# https://data.services.jetbrains.com/products?code=PCP,PCC&release.type=release&fields=name,releases
+
+HOMEPAGE="https://www.jetbrains.com/pycharm/"
+SRC_URI="
+	amd64? (
+		https://download.jetbrains.com/python/${P}.tar.gz
+	)
+	arm64? (
+		https://download.jetbrains.com/python/${P}-aarch64.tar.gz
+	)
+	x86? (
+		https://download.jetbrains.com/python/${P}.tar.gz
+	)
+"
+
+if [[ ${PN} == *-professional ]]; then
+	S="${WORKDIR}/${PN/%-professional/}-${PV}"
+	LICENSE="|| ( JetBrains-business JetBrains-classroom JetBrains-educational JetBrains-individual )
+	Apache-2.0 BSD BSD-2 CC0-1.0 CC-BY-2.5 CC-BY-3.0 CC-BY-4.0 CPL-1.0 CDDL CDDL-1.1 EPL-1.0 EPL-2.0
+	GPL-2 GPL-2-with-classpath-exception ISC JDOM LGPL-2.1 LGPL-3 MIT MPL-1.1 MPL-2.0 OFL-1.1
+	PYTHON Unicode-DFS-2016 Unlicense UPL-1.0 ZLIB"
+else
+	LICENSE="|| ( JetBrains-business JetBrains-classroom JetBrains-educational JetBrains-individual )
+	Apache-2.0 BSD BSD-2 CC0-1.0 CC-BY-2.5 CC-BY-3.0 CPL-1.0 CDDL-1.1 EPL-1.0 GPL-2
+	GPL-2-with-classpath-exception ISC JDOM JSON LGPL-2+ LGPL-2.1 LGPL-3 MIT MPL-1.1 MPL-2.0
+	OFL-1.1 UPL-1.0 ZLIB"
+fi
+
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm64 ~x86"
+IUSE="+bundled-jdk"
+
+if [[ ${PN} == *-professional ]]; then
+	IUSE+=" +bundled-xvfb"
+fi
+
+BDEPEND="dev-util/debugedit
+	dev-util/patchelf
+"
+
+# NOTE
+#  The remote-dev-server present in pycharm-professional contains most of the
+#  libraries need for the bundled jdk. These are not in the RUNPATH of the jdk.
+#  So the dependencies are actually needed.
+RDEPEND="
+	!bundled-jdk? (
+		>=virtual/jre-1.8
```

Comment:
I tested this, both pycharms require `>=virtual/jre-17:*`

Also please check the [java documentation](https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE) on how to specify java dependencies correctly.

Reply:
> I tested this, both pycharms require `>=virtual/jre-17:*`
> 
> Also please check the [java documentation](https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE) on how to specify java dependencies correctly.

Another one that I forgot to add, very true
This primarily closes #907845.